### DISCUSSION TITLE
Read model that reads another model

### DISCUSSION THREAD

**Discussion by dmuczynski** [boostercloud/booster#950] [Q&A]
I have created a read model that has the following setup:

`public id: UUID,
@sequencedBy readonly timestamp: TimeKey,
readonly hashtags: Array<string>,`

where hashtags are an array of hashtagIds.

When I query this object, I would then like to query the HashtagRead model to populate the hashtags with the entities, so it will be something like this:

`public id: UUID,
@sequencedBy readonly timestamp: TimeKey,
readonly hashtags: Array<Hashtab>,`

Is this possible?

**Comment by NickSeagull** [comment]
Closing as stale, we now can create async getters that do this
